test/parallel/test-dgram-send-callback-buffer-empty-address.js

```diff
@@ -14,4 +14,4 @@ const onMessage = common.mustCall(function(err, bytes) {
   client.close();
 });
 
-client.send(buf, common.PORT, onMessage);
+client.bind(0, () => client.send(buf, client.address().port, onMessage));
```

Comment:
Yeah I also find `common.mustCall()` a bit redundant here which is the reason why I didn't suggest it in the first place.